run

```diff
@@ -469,7 +469,8 @@ lint() {
       # Fix python black code formatting issues, run:
       run_command black ${DIR_TO_RUN} || exit_code=1
       run_command codespell -w -i 3 ${DIR_TO_RUN} --ignore-words codespell_ignore_words.txt \
-                            --skip="*.onnx,*.min.js,*.min.js.map,Contrastive_learning_Notebook.ipynb,./data" || exit_code=1
+                            --skip="*.onnx,*.min.js,*.min.js.map,Contrastive_learning_Notebook.ipynb,./data" \
+                            --skip="./applications/nvidia_nim/nvidia_nim_nvclip/nvidia_nim.yaml" || exit_code=1
```

Comment:
Could you elaborate on this change? Would be good to move images to `.png` or similar instead of `.yaml` fields

Reply:
Using base64 encoded images is because it avoids broken links if the images were removed. Plus, the API takes base64 encoded images.  The other option is to move them into a separate file.

Reply:
Moving the sample encoded images to a separate YAML file also triggered codespell errors.

Reply:
Please see below on using a config file to consolidate the ignored files in one place.

Reply:
Removed.